app/repositories/DefaultValues.scala

```diff
@@ -49,4 +56,15 @@ object DefaultValues extends Logging {
         NotFound(s"Could not find endpoint in Dynamic Stub matching the URI: ${request.uri}")
     }
   }
+
+  def getDefaultRequestHandler(url: String): Future[Result] = {
+    dataRepository
+      .find(equal("_id", url), equal("method", GET))
+      .map {
+        case stubData@Some(_: DataModel) =>
+          Status(stubData.head.status)(stubData.head.response.get)
+        case None =>
+          NotFound(s"Failed to find the default API endpoint in the repository matching the URI: $url")
```

Comment:
You can extract required values in a safe way:

`      .map {
        case Some(DataModel(_, _, _, status, Some(response))) =>
          Status(status)(response)
        case None =>
          NotFound(s"Failed to find the default API endpoint in the repository matching the URI: $url")
      }`